msal/src/main/java/com/microsoft/identity/client/internal/controllers/LocalMSALController.java

```diff
@@ -198,26 +204,58 @@ void throwIfNetworkNotAvailable(final Context context) throws MsalClientExceptio
                     "Device network connection is not available."
             );
         }
+
+        Logger.info(
+                TAG + methodName,
+                "Network status: connected"
+        );
     }
 
     private ICacheRecord saveTokens(final OAuth2Strategy strategy,
                                     final AuthorizationRequest request,
                                     final TokenResponse tokenResponse,
                                     final OAuth2TokenCache tokenCache) throws ClientException {
+        final String methodName = ":saveTokens";
+        Logger.verbose(
+                TAG + methodName,
+                "Saving tokens..."
+        );
         return tokenCache.save(strategy, request, tokenResponse);
     }
 
     @Override
     public void completeAcquireToken(final int requestCode,
                                      final int resultCode,
                                      final Intent data) {
+        final String methodName = ":completeAcquireToken";
+        Logger.verbose(
+                TAG + methodName,
+                "Completing acquire token..."
+        );
         mAuthorizationStrategy.completeAuthorization(requestCode, resultCode, data);
     }
 
     @Override
     public AcquireTokenResult acquireTokenSilent(
             final MSALAcquireTokenSilentOperationParameters parameters)
             throws MsalClientException, IOException, ClientException, MsalArgumentException, MsalUiRequiredException {
+        final String methodName = ":acquireTokenSilent";
+        Logger.verbose(
+                TAG + methodName,
+                "Acquiring token silently..."
+        );
+
+        final List<String> msalScopes = new ArrayList<>();
```

Comment:
@addev-ashish Calling out a subtle bugfix here....

Since tokens written by v1 don't contain real scopes (resources), `openid`, `profile`, and `offline_access` are added if they are not present in the request